build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+go build -o build/api main.go
```

Comment:
This may break if folder `build` doesn't exist. I recommend the full file content as:
Bash scripts often using `#!/bin/bash` directly, rather then `#!/usr/bin/env bash`.

```sh
#!/bin/bash

mkdir -p build
go build -o build/server main.go
```

Reply:
And don't forget `chmod a+rx build.sh` before `git add` this file.

Reply:
Please also add a `.gitignore` file to ignore the `build` folder.